driver/CMakeLists.txt

```diff
@@ -2,7 +2,7 @@
 ADD_SUBDIRECTORY( src )
 
 #################################################################################
-
+INCLUDE_DIRECTORIES(${PROJECT_SOURCE_DIR} ${CMAKE_CURRENT_SOURCE_DIR}/src)

```

Comment:
PROJECT_SOURCE_DIR is for getting includes like `libendian/src/*` and the other one for the base driver files like the interface